config.json

```diff
@@ -13,6 +13,16 @@
       "topics": [
       ]
     },
+    {
+      "uuid": "f750e380-fe16-42c0-8b3d-8acd7e443a6e",
+      "slug": "protein-translation",
```

Comment:
I wouldn't put this between hello-world and leap because leap seems easier since don't have to deal with lists or Maybe. So at the very least, after `leap`.